python_modules/dagster/dagster/core/execution/context/system.py

```diff
@@ -493,14 +493,19 @@ def for_input_manager(
         source_handle: Optional[StepOutputHandle] = None,
         resource_config: Any = None,
         resources: Optional["Resources"] = None,
+        artificial_output_context: Optional["OutputContext"] = None,
```

Comment:
Not a change I'm suggesting making in this PR, but I wonder if we should stop including an artificial output context in these situations.

Reply:
agreed, right now it's kinda an olive branch for people that are used to writing non-asset specific io managers, but I don't think it's a good idea to keep around forever.